### ISSUE TITLE
[Feature Request]: Play now

### ISSUE THREAD

**Issue by greg321321** [tubearchivist/tubearchivist#603]
### I've read the documentation

- [X] I have read the [how to open an issue](https://github.com/tubearchivist/tubearchivist/blob/master/CONTRIBUTING.md#how-to-open-an-issue) guide, particularly the [feature request](https://github.com/tubearchivist/tubearchivist/blob/master/CONTRIBUTING.md#feature-request) section.

### Your Feature Request

It would be nice to have an easy way to play a youtube video right away.

Use case: I search the internet for say an instruction on how to repair my dishwasher. Typically the first hit is a youtube video. I want to play the video right away from tube archivist.

Suggested solution: The chrome plugin allows us to mark the video for download. Once marked, we see a checkmark icon on the injected code button. When I click on that icon for a second time, I could be brought to a tube archivist page that attempts to play the video right away. I wouldn't mind seeing a status on this page that says something like 'please wait while your video is being prepared'. 

That would be a lot better than the 4 ads I had to sit through to watch a 5 minute instructable video on youtube.

### Your help is needed!

- [X] Yes I will work on this in the next few days or weeks.

**Comment by bbilly1** [comment]
We already have a browser extension Tube Archivist Companion, that does basically that:

https://github.com/tubearchivist/browser-extension

**Comment by greg321321** [comment]
I am talking about the browser extension. It is too many clicks. To immediately view the video I need to:

1. Click on browser extension "download" button
2. Goto my bookmarks, click on link to tubearchivist
3. click on search icon
4. click on search field
5. type in search keywords
6. scroll through results to find video
7. if I don't find it, maybe wait a bit for it to finish downloading, then try to find it again.
8. Once I find the video, I click play and am good.

Vs what I am proposing:
1. Click on browser extension "download" button
2. Click on the same button again
3. I am automatically brought to tube archivist page for that video.
4. If video is not done downloading, I am presented with a message.
5. When video is ready, it is presented on screen
6. I click play and am good.

**Comment by bbilly1** [comment]
The extension will build a `Open in TA` button instead of a download button if the video is already downloaded. You might have to refresh the YT page to see it.

**Comment by greg321321** [comment]
ok that's not bad. I also took a deeper look at some of the sort capability of tube archivist. If I set my home page sort to 'sort by date downloaded', then the video will appear first in that list once its dowload is complete. It's better to wait in tube archivist for the download to complete since tube archivist will not attempt to show me more advertisements while I wait. :)
